src/components/HomeMarketSliders.tsx

```diff
@@ -0,0 +1,814 @@
+import { FC, useCallback, useContext, useEffect, useMemo } from 'react';
```

Comment:
Please remove unused Class. If you are intending to use it in the future then disregard this comment. :)